src/names.jl

```diff
@@ -152,3 +152,43 @@ function named_axiskeys(A::AbstractArray)
     NT = NamedTuple{dimnames(A)}
     NT(axiskeys(A))
 end
+
+
+"""
```

Comment:
needs a function signature no?

Reply:
That doesn't seem to be the convention used in most of the other docstrings for this package. Since the allowed types of the arguments are pretty broad I'm also not sure how helpful that would be. The array could be a `KeyedArray` or `NamedDimsArray(KeyedArray(...))` and the other arguments could be a `Tuple`, `Pair{<:Integer, <:AbstractVector}`, `Pair{<:Symbol, <:AbstractVector}` or `Pair{Symbol, <:Pair}`.

Reply:
if it's not convention that's fair enough